yang/frr-bgp.yang

```diff
@@ -526,6 +530,66 @@ module frr-bgp {
     uses global-afi-safi-vpn-network-config;
   }
 
+  augment "/frr-rt:routing/frr-rt:control-plane-protocols/frr-rt:control-plane-protocol/bgp/global/afi-safis/afi-safi/l2vpn-evpn" {
+    container advertise {
```

Comment:
Why defined a container for these knobs. They could directly appear under the l2vpn-evpn afi-safi

Reply:
done!